### ISSUE TITLE
Add support for large (>8Mb) PNG files

### ISSUE THREAD

**Issue by kDanil** [kleisauke/net-vips#55]
Hello,

NetVips 1.1.0
NetVips.Native 8.8.3

When I try to process large (>8Mb) PNG file on Linux I'm receiving warnings 
```
(NetVips:1): VIPS-WARNING **: 11:11:46.164: iTXt: chunk data is too large
```
and exception
```
NetVips.VipsException: unable to call VipsForeignLoadPngBuffer
vips_realpath: unable to form filename
unix error: No such file or directory
```

I did a research and found out that it is limitation of libpng library. Also it seems libvips is supporting a way how to increase a limit.
https://github.com/libvips/libvips/blob/ecb22cdecc98c56eac28bb4b10ebcb825a60791a/libvips/foreign/vipspng.c#L305

However, apparently there is no way to control it from .NET code.
Could it be enabled by default?

**Comment by kleisauke** [comment]
You've run into libvips/libvips#1424, which is fixed in libvips v8.9.0. Updating to [NetVips.Native v8.9.0-rc4](https://www.nuget.org/packages/NetVips.Native/8.9.0-rc4) (pre-release) should fix this.

I'll try to pack the final version of libvips 8.9.0 either this weekend or in the course of next week.

**Comment by kleisauke** [comment]
[NetVips v1.2.0](https://www.nuget.org/packages/NetVips/1.2.0) and [NetVips.Native v8.9.1](https://www.nuget.org/packages/NetVips.Native/8.9.1) is now available.
